misc/assets/inforequests.svg

```diff
@@ -0,0 +1,248 @@
+  </g>
+  <g transform="translate(100 200)">
+    <use href="#model-narrow" />
+    <text class="caption">Inforequest</text>
+    <path class="relation from-optional to-many" d="M 0,20 0,180" />
+    <path class="relation from-one to-many" d="M 20,-20 20,-40 450,-40 450,-80" />
+    <path class="relation from-one to-many" d="M 50,0 450,0" />
+    <text class="note" x="430" y="0">1+</text>
+  </g>
+  <g transform="translate(150 400)">
+    <use href="#model" class="external" />
+    <text class="caption">E-mail Message</text>
+    <path class="relation from-optional to-optional" d="M 100,0 400,0" />
+  </g>
+  <g transform="translate(300 100)">
+    <use href="#model" />
+    <text class="caption">Inforequest Draft</text>
+    <path class="relation from-many to-optional" d="M 0,-20 0,-40 500,-40 500,-20" />
+  </g>
+  <g transform="translate(300 300)">
+    <use href="#model" />
+    <text class="caption">Inforequest E-mail</text>
+    <path class="relation from-one to-link_attribute" d="M -100,0 -200,0" />
+  </g>
+  <g transform="translate(600 100)">
+    <use href="#model" />
+    <text class="caption">Action Draft</text>
+    <path class="relation from-many to-optional" d="M 0,20 0,80" />
+    <path class="relation from-many to-many" d="M 100,0 150,0" />
+  </g>
+  <g transform="translate(600 200)">
+    <use href="#model-narrow" />
+    <text class="caption">Branch</text>
+    <path class="relation from-many to-one" d="M 25,-20 25,-60 175,-60 175,-80" />
+    <path class="relation from-many to-one" d="M 50,0 100,0" />
+    <path class="relation from-one to-many" d="M -25,20 -25,180" />
+    <text class="note" x="-10" y="180">1+</text>
+    <text class="note left-rotation" x="-135" y="-30">Contained</text>
+    <path class="relation from-many to-optional" d="M 25,20 25,180" />
+    <text class="note right-rotation" x="70" y="-30">Advanced by</text>
+  </g>
+  <g transform="translate(600 400)">
+    <use href="#model-narrow" />
+    <text class="caption">Action</text>
+  </g>
+  <g transform="translate(800 100)">
+    <use href="#model-narrow" class="external" />
+    <text class="caption">Obligee</text>
+    <path class="external relation from-one to-many" d="M 25,20 25,80" />
+  </g>
+  <g transform="translate(800 200)">
+    <use href="#model" class="external" />
+    <text class="caption">Historical Obligee</text>
+  </g>
+
+  <g transform="translate(150 500)">
+    <use href="#model" class="external" />
+    <text class="caption">Attachment</text>
+  </g>
+  <g transform="translate(350 550)">
+    <use href="#model" />
+    <text class="caption">Inforequest Draft</text>
+    <path class="relation from-optional to-many" d="M 0,-20 0,-50 -100,-50" />
+  </g>
+  <g transform="translate(600 550)">
+    <use href="#model" />
+    <text class="caption">Action Draft</text>
+    <path class="relation from-optional to-many" d="M 0,-20 0,-50 -350,-50" />
+  </g>
+  <g transform="translate(800 550)">
```

Comment:
Pls dodrzuj zauzivane vertikalne vzdilenosti medzi modelmi. A pouzi 600 nie 550.

Reply:
@martinmacko47 Medzi itemy `ApplicantAction`, `Request`, `ClarificationResponse`, `Appeal` som dal vertikalnu vzdialenost na `75` pre lepsi vziualny efekt. Upravit tiez na `100`?
![Screenshot from 2019-09-03 12-43-04](https://user-images.githubusercontent.com/29918752/64166936-748ee600-ce48-11e9-9644-4ba0d628f15a.png)

Reply:
Ok. Tu kludne nechaj 75. Len pri relacii k attachmentu si tiez dal uzsiu vertikalnu vzdialenost a vyzeralo to moc natlacene na seba.

Reply:
:ok: